src/components/warningbox.js

```diff
@@ -0,0 +1,138 @@
+                            }}
+                            onClick = {onClick}
+                        >
+                            <svg
+                                height  = '25px'
+                                width   = '25px'
+                                viewBox = '0 0 100 100'
+                            >
+                                <path 
+                                    fillRule = 'evenodd'
+                                    clipRule = 'evenodd'
+                                    fill     = 'red'
+                                    d        = 'M73.9,5.75c0.467-0.467,1.067-0.7,1.8-0.7c0.7,0,1.283,0.233,1.75,0.7l16.8,16.8  c0.467,0.5,0.7,1.084,0.7,1.75c0,0.733-0.233,1.334-0.7,1.801L70.35,50l23.9,23.95c0.5,0.467,0.75,1.066,0.75,1.8  c0,0.667-0.25,1.25-0.75,1.75l-16.8,16.75c-0.534,0.467-1.117,0.7-1.75,0.7s-1.233-0.233-1.8-0.7L50,70.351L26.1,94.25  c-0.567,0.467-1.167,0.7-1.8,0.7c-0.667,0-1.283-0.233-1.85-0.7L5.75,77.5C5.25,77,5,76.417,5,75.75c0-0.733,0.25-1.333,0.75-1.8  L29.65,50L5.75,26.101C5.25,25.667,5,25.066,5,24.3c0-0.666,0.25-1.25,0.75-1.75l16.8-16.8c0.467-0.467,1.05-0.7,1.75-0.7  c0.733,0,1.333,0.233,1.8,0.7L50,29.65L73.9,5.75z'
+                                />
+                            </svg>
+                        </div>
+                    </div>
+                </span>
+                <span
+                    style = {{
+                        display       : 'inline-block',
+                        height        : '60px',
+                        width         : 'calc(100% - 60px)',
+                        margin        : 0,
+                        overflow      : 'hidden',
+                        verticalAlign : 'top',
+                        position      : 'absolute',
+                        pointerEvents : 'none'
+                    }}
+                    onClick = {onClick}  
+                >
+                    { 
+                        error ?
+                        <p
+                            style = {{
+                                color          : 'red',
+                                fontSize       : '12px',
+                                position       : 'absolute',
+                                width          : 'calc(100% - 60px)',
+                                height         : '60px',
+                                boxSizing      : 'border-box',
+                                margin         : 0,
+                                padding        : 0,
+                                paddingRight   : '10px',
+                                overflowWrap   : 'break-word',
+                                display        : 'flex',
+                                flexDirection  : 'column',
+                                justifyContent : 'center',
+                                ...style.message
+                            }}
+                        >
+                            {format(locale.format, error)}
```

Comment:
I think it would be awesome to be able to just pass the locale key and its parameters to this components, and it would do all the formatting, to any extend in which we would no longer need to import the format() function in the main component.
But I think for now this is fine as-is and we can revisit the locale system once we need to.